Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM alpine
+
+RUN apk add --no-cache openssl curl && \
+    cd /usr/local/bin && \
+    curl -s -LO https://storage.googleapis.com/kubernetes-release/release/`curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt`/bin/linux/amd64/kubectl && \
```

Comment:
probably need to use a fixed version of kubernetes release 
and ENV variable with K8S version

to be sure that everytime we build the container we have the same behavior

Reply:
Agree, thanks.